assets/js/blocks/reviews/frontend-block.tsx

```diff
@@ -2,23 +2,43 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import PropTypes from 'prop-types';
 import { getSetting } from '@woocommerce/settings';
 import LoadMoreButton from '@woocommerce/base-components/load-more-button';
 import {
 	ReviewList,
 	ReviewSortSelect,
 } from '@woocommerce/base-components/reviews';
 import withReviews from '@woocommerce/base-hocs/with-reviews';
+import type { ChangeEventHandler, MouseEventHandler } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { ReviewListAttributes } from '../../base/components/reviews/review-list';
+import { Review } from '../../base/components/reviews/types';
+
+type FrontendBlockAttributes = ReviewListAttributes & {
+	showOrderby?: 'false' | 'true';
```

Comment:
These also I couldn't decide should they be `?` or not

Reply:
I think the `showOrderby` and `showLoadMore` attributes aren't prepared properly, leading them to hold string values (`'true'` and `'false'`) when the block is rendered on the frontend. Those attributes are typed as boolean according to `assets/js/blocks/reviews/attributes.js`.

IMO, we should update `assets/js/blocks/reviews/frontend.js` to process the value of those attributes to match their registered types.

Reply:
You can take the Mini Cart block as [an example](https://github.com/woocommerce/woocommerce-blocks/blob/6248528985fde1900a1631cb7c74fe00b4eb6f2e/assets/js/blocks/mini-cart/component-frontend.tsx#L47).